# point2D msg for Lux

int16  x
int16  y
